common/src/main/kotlin/fr/agrouagrou/common/player/PlayerRegistry.kt

```diff
@@ -0,0 +1,19 @@
+package fr.agrouagrou.common.player
+
+import java.util.*
+
+class PlayerRegistry {
+    private val mutablePlayers = mutableMapOf<UUID, Player>()
+    val players = mutablePlayers as Map<UUID, Player>
+
+    fun register(username: String): Player {
+        val player = Player(username)
```

Comment:
How does the host know which player is which if you use a generated ID and not check for duplicate usernames ?

Reply:
Good point, I thought I had the ID in the reply, but apparently I removed it at some point. I'll add that back to the reply.

Reply:
Just checked, but the player ID is in fact already sent back to the client.